wallet/db.c

```diff
@@ -949,6 +949,11 @@ static struct migration dbmigrations[] = {
     {NULL, migrate_invalid_last_tx_psbts},
     {SQL("ALTER TABLE channels ADD channel_type BLOB DEFAULT NULL;"), NULL},
     {NULL, migrate_fill_in_channel_type},
+    {SQL("ALTER TABLE channels ADD remote_feerate_base INTEGER;"), NULL},
+    {SQL("ALTER TABLE channels ADD remote_feerate_ppm INTEGER;"), NULL},
+    {SQL("ALTER TABLE channels ADD remote_cltv_expiry_delta INTEGER;"), NULL},
+    {SQL("ALTER TABLE channels ADD remote_htlc_maximum_msat BIGINT;"), NULL},
+    {SQL("ALTER TABLE channels ADD remote_htlc_minimum_msat BIGINT;"), NULL},
```

Comment:
Ah, SQL is hard! Thinking that in this case, it is better to put this inside a procedure as we did with `migrate_fill_in_channel_type` and put all migration between a `BEGIN TRANSACTION ..... COMMIT`?

Reply:
The migration code _is_ wrapped in a transaction already.